### DISCUSSION TITLE
Installation on M1 MacAir OS Monterey 12.6.9 does not work

### DISCUSSION THREAD

**Discussion by cassiopeiait** [iraf-community/iraf#357] [Q&A]
When I try to run 

make 2>&1 | tee build.log

I get an error, I attach it here. I have macOS Monterey 12.6.9, M1 Mac Air
I have used iraf-2.17.1

These are the last lines of the error log:

```

cc  -g -O2 -DHAVE_CFITSIO -I../cfitsio   -c -o votStack.o votStack.c
cc  -g -O2 -DHAVE_CFITSIO -I../cfitsio   -c -o votHandle.o votHandle.c
/Users/tanja/iraf-2.17.1/unix/bin.macos64/xc.e -c votUtil_spp.x -o votUtil_spp.o
make[2]: *** [votUtil_spp.o] Killed: 9
make[1]: *** [libvotable] Error 2
make: *** [vendor] Error 2

```

Any help would be appreciated!

[build.log](https://github.com/iraf-community/iraf/files/13825034/build.log)

**Comment by olebole** [comment]
I have no real idea here: signal 9 means that the program was hard-killed from another program.
Googling this on the internet, it may help to try again, maybe after rebooting the computer, and/or to re-install xCode.
See also 

https://apple.stackexchange.com/questions/258623/how-to-fix-killed-9-error-in-mac-os
https://developer.apple.com/forums/thread/701242

for more hints.

**Comment by olebole** [comment]
Not sure whether you had success in the meantime. If not, you could try out the [macOS installation procedure](https://iraf-community.github.io/install.html#macos).
